oadm.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"os/exec"
+)
+
+// OadmData is a task factory for tasks that fetch the output of oc admin diagnostics.
```

Comment:
Isn't it "oadm diagnostics"?

I didn't know, but these two seem to produce the same output:

```
$ oc adm diagnostics
$ oadm diagnostics
```

I don't know what the openshift-client package on RHEL installs, but I think it would be reasonable to use the `oc adm` version so that we have a binary dependency on a single name.

If we use `oadm`, we should document that new dependency.